app/views/external_users/claims/offence_details/_default_fields.html.haml

```diff
@@ -1,25 +1,18 @@
 - locale_scope = 'external_users.claims.offence_details.fields'
-#cc-offence.form-group{ class: error_class?(@error_presenter, :offence_category) ? 'form-group-error' : '' }
-  %a#offence_category
-  %label.form-label-bold{ for: 'claim_offence_category_description' }
```

Comment:
`claim_offence_category_description` is this identifier still required in `features/page_objects/claim_form_page.rb`

Reply:
removed as suggested in `cleanup dead codes` push